src/main/osd/osd_elements.c

```diff
@@ -1639,6 +1660,7 @@ const osdElementDrawFn osdElementDrawFunction[OSD_ITEM_COUNT] = {
 #ifdef USE_GPS
     [OSD_HOME_DIR]                = osdElementGpsHomeDirection,
     [OSD_HOME_DIST]               = osdElementGpsHomeDistance,
+    [OSD_EFFICIENCY]              = osdElementEfficiency,
```

Comment:
I thought I mentioned this before but can't find the comment now. Please add this to the end of the definition (including the `#ifdef USE_GPS` block). We prefer to keep this array defined in the same order as `osd_items_e` in osd.h. Since they are related arrays this helps to make it easier to ensure they don't get out of sync.